assignment2.txt

```diff
@@ -0,0 +1,14 @@
+Ava N.
+June 29, 2016
+
+1.  There can be two or more operators in a single line of program code as long as there is a logical operator seperating the relational operator statements. 
+
+2. Syntax errors are incorrect spellings of reserved words or incorrect puncuation. Logical errors give an output that is unexpected or unwanted. 
+
+3. 10>=10 && 10<25 && 10 !=12
```

Comment:
Correct, but remember that boolean values don't exist in C! So the answer is actually 1!